test/e2e/quick_start_test.go

```diff
@@ -34,3 +46,16 @@ var _ = Describe("ClusterClass Creation using Cluster API quick-start test [PR-B
 		}
 	})
 })
+
+var _ = Describe("Cluster creation with [Ignition] bootstrap", func() {
```

Comment:
We should probably keep this test PR blocking

Reply:
I was wondering about that specifically - in core CAPI we only have one release blocking test, here we have three. Was wondering if we should rethink that. My preference would be for speed, but I don't hold that strongly.

Reply:
I think ignition is a bit more tricky in infra providers. How much difference does it make? What concurrency do we use?

I assume if we use concurrency/nodes >= 3 it shouldn't make a difference

Reply:
(in CAPI the runtimesdk test takes 19 min vs quick start 10m IIRC)

Reply:
To answer my own question. Looks like we're running everything sequentially o.O. Let's see if our e2e tests are able to run in parallel: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2340 (if they don't I'll open a follow-up to implement it)